analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DoNotLockOnSharedResource.CSharp11.cs

```diff
@@ -5,13 +5,11 @@ namespace Tests.Diagnostics
 {
     public class LockOnThisOrType
     {
-        public void MyLockingMethod()
+        public void RawStringLiterals()
         {
```

Comment:
DoNotLockWeakIdentityObjects is missing similar test cases with raw string literals.